packages/glimmer-compiler/lib/template-compiler.ts

```diff
@@ -366,7 +378,23 @@ function assertValidYield({ hash }): string {
   }
 }
 
-function assertValidHasBlock({ params }): string {
+function assertValidPartial({ params, hash, escaped }) /* : expr */ {
+  if (params && params.length !== 1) {
+    throw new Error(`Partial found with no arguments. You must specify a template name.`);
+  } else if (hash && hash.pairs.length > 0) {
+    throw new Error(`partial does not take any named arguments`);
+  } else if (!escaped) {
+    throw new Error(`{{{partial ...}}} is not supported, please use {{partial ...}} instead`);
+  }
+
+  return params;
+}
+
+function assertValidHasBlock({ params, hash }): string {
```

Comment:
This seems identical to the `assertValidHasBlockParams` function just below.  What am I missing?